### DISCUSSION TITLE
Change prediction `score_threshold` and `merge_threshold` during `rastervision predict` using pipeline

### DISCUSSION THREAD

**Discussion by aerotractjack** [azavea/raster-vision#1778] [Q&A]
Hello, I am using the RV pipeline for my object detection task and have encountered an issue. It is not a bug but maybe a feature to implement in the future. 

You can view my config [in this repo](https://github.com/aerotractjack/rvml/blob/2docker/train.py) 
Python 3.9
PopOS 22.04

When using a model with the pipeline, you have to set the prediction options, `merge_thresh` and `score_thresh` before training. Once training finishes and its time to predict, I can't find a way of modifying the pipeline to use a different value for my `score_thresh`. I am currently predicting via the CLI like `rastervision predict ...`, but looked into the `Predictor` class and didn't see anything that would allow me to do this there either. My only idea for now is to modify the pipeline config, but trying that once led to a `file unreadable` error. Is this possible with my current setup?

**Comment by AdeelH** [comment]
Sorry for the very late reply. Yes, you are right that there is no way to do this using the `predict` CLI command. It should be safe to manually edit the pipeline config though, as long as you don't mess up the JSON format.

Another workaround is to modify your `get_config()` to add the new scenes you want to predict on to the validation or test set in your `DatasetConfig`, then modify those params, and then run only the `predict` stage of the pipeline like so:

```sh
rastervision run inprocess "your_config_file.py" predict
```
